docs/twig-components/method/dump.md

```diff
@@ -1,6 +1,16 @@
 # dump
 
-`dump()` is a Twig function to ...
+`dump()` is a Twig function to dump the entire object, similar to PHP's var_dump.
 
+```twig
+{% set about = content('page', {'slug': 'about'}) %}
 
-Source: Bolt
\ No newline at end of file
+{{ dump(about) }}
+```
+
+For more info on debugging your Bolt site, see the chapter on [Debugging Bolt](https://docs.bolt.cm/4.0/debugging).
```

Comment:
@mcdennem I think since this PR is going to be merged into 5.0, links should be updated to link to 5.0.
https://docs.bolt.cm/4.0/debugging => https://docs.bolt.cm/5.0/debugging